<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="6723" started-at="2018-12-27T09:53:03Z" finished-at="2018-12-27T09:53:10Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="6723" started-at="2018-12-27T09:53:03Z" finished-at="2018-12-27T09:53:10Z">
      <class name="ReadExcelData.Testng">
        <test-method status="PASS" signature="brftest()[pri:0, instance:ReadExcelData.Testng@1593948d]" name="brftest" is-config="true" duration-ms="6694" started-at="2018-12-27T15:23:03Z" finished-at="2018-12-27T15:23:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- brftest -->
        <test-method status="FAIL" signature="login(java.lang.String, java.lang.String)[pri:0, instance:ReadExcelData.Testng@1593948d]" name="login" duration-ms="0" started-at="2018-12-27T15:23:10Z" data-provider="Authentication" finished-at="2018-12-27T15:23:10Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Method public void ReadExcelData.Testng.login(java.lang.String,java.lang.String) requires a @DataProvider named : Authentication in class ReadExcelData.DataProvidertest]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Method public void ReadExcelData.Testng.login(java.lang.String,java.lang.String) requires a @DataProvider named : Authentication in class ReadExcelData.DataProvidertest
	at org.testng.internal.Parameters.findDataProvider(Parameters.java:336)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:498)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1308)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1036)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1126)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="aftrtest()[pri:0, instance:ReadExcelData.Testng@1593948d]" name="aftrtest" is-config="true" duration-ms="0" started-at="2018-12-27T15:23:10Z" finished-at="2018-12-27T15:23:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- aftrtest -->
      </class> <!-- ReadExcelData.Testng -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
